deployments/app-land/resources/uservice-ghslacker.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: uservice-ghslacker
```

Comment:
So this is maybe a controversial opinion, but I'm kind of hoping we can drop the ``uservice-`` and ``u-`` prefixes in the Roundtable era. Aesthetically they're a bit ugly and give me a hard time to feel good about the branding of projects.

I think the original reason the `uservice-` prefix got used was to help make it easier to find microservice repositories on GitHub, but nowadays I think we could leverage the [roundtable](https://github.com/search?q=topic%3Aroundtable+org%3Alsst-sqre) GitHub topic.